bigbone/src/test/kotlin/social/bigbone/api/method/SuggestionMethodsTest.kt

```diff
@@ -0,0 +1,39 @@
+package social.bigbone.api.method
+
+import org.amshove.kluent.shouldBeEqualTo
+import org.amshove.kluent.shouldHaveSize
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import social.bigbone.api.exception.BigBoneRequestException
+import social.bigbone.testtool.MockClient
+
+class SuggestionMethodsTest {
+
+    @Test
+    fun getSuggestions() {
+        val client = MockClient.mock("suggestions.json")
+        val suggestionMethods = SuggestionMethods(client)
+        val suggestions = suggestionMethods.getSuggestions(2).execute()
+        suggestions shouldHaveSize 2
+        suggestions[0].source.name.lowercase() shouldBeEqualTo "past_interactions"
+        suggestions[1].source.name.lowercase() shouldBeEqualTo "global"
```

Comment:
It’s usually best to compare as raw a data point as you can without any transformations applied, so I’d suggest to instead expect the enum values here:

```suggestion
        suggestions[0].source shouldBeEqualTo SourceSuggestion.PAST_INTERACTIONS
        suggestions[1].source shouldBeEqualTo SourceSuggestion.GLOBAL
```